debugging/debugging_session_displacement.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
For now I just added this here, but I wonder if it could be a useful pattern. For debugging code, we could place in this directory. That way it is kept for the long-term and easily accessible, but is not mixed with the main codebase. WDYT?

Reply:
Not sure what I was thinking here!